flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisProducer.java

```diff
@@ -57,6 +55,9 @@
 	/** Properties to parametrize settings such as AWS service region, access key etc. */
 	private final Properties configProps;
 
+	/** Configuration for KinesisProducer. */
+	private final KinesisProducerConfiguration producerConfig;
```

Comment:
On second thought: do we actually really need to have a field for this?
Or can we just instantiate in in the method (it doesn't seem to be used across different methods)?